torch/csrc/dynamo/eval_frame.c

```diff
@@ -304,10 +305,49 @@ typedef struct cache_entry {
   struct cache_entry* next;
 } CacheEntry;
 
+#define Py_NONE_CACHE_ENTRY (CacheEntry*)Py_None
+
+static void destroy_cache_entry(CacheEntry* e);
```

Comment:
You don't need this anymore right? You can just Py_DECREF now

Reply:
This is the destructor of CacheEntry object. This will be called when ref count is 0. So, we do need this.